app/app.go

```diff
@@ -187,7 +187,7 @@ var (
 	maccPerms = map[string][]string{
 		authtypes.FeeCollectorName:      nil,
 		distrtypes.ModuleName:           nil,
-		minttypes.ModuleName:            {authtypes.Minter},
+		minttypes.ModuleName:            {authtypes.Minter, authtypes.Burner},
```

Comment:
nit: add comment why we need this (it's the result of a confusing mint mechanism iirc)

Reply:
done